jumpscale/clients/stellar/stellar.py

```diff
@@ -835,22 +836,24 @@ def get_asset(self, code="TFT", issuer=None) -> stellar_sdk.Asset:
             asset_issuer = _NETWORK_KNOWN_TRUSTS[network].get(code, None)
             return Asset(code, asset_issuer)
 
-    def place_sell_order(
+    def manage_sell_order(
```

Comment:
Let's create more helpers 
e.g (cancel_sell_order or delete_sell_order), better imo to pass different amount as flag for deletion

Reply:
Done and testd, place_sell_order and cancel_sell_order